flink-table/flink-sql-parser/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -190,6 +190,22 @@ SqlShowTables SqlShowTables() :
     }
 }
 
+SqlRichDescribeTable SqlRichDescribeTable() :
```

Comment:
minor: should this be `SqlDescribeExtendedTable`?

Reply:
It should be probably just SQLDescribeTable. Whether it's for "rich" or "extended" is up to the option "isExtended".

However, the corresponding class description has an explanation for this, which seems fine.